runtime/port/zos390/protect_helpers.c

```diff
@@ -25,6 +25,8 @@
 #include <sys/mman.h>
 #include <errno.h>
 
+extern intptr_t _MPROT(uintptr_t address, uintptr_t length);
+extern intptr_t _MUNPROT(uintptr_t address, uintptr_t length);
```

Comment:
Please add comments clarifying where these functions are defined.
```c
extern intptr_t _MPROT(uintptr_t address, uintptr_t length); /* j9mprotect.s */
extern intptr_t _MUNPROT(uintptr_t address, uintptr_t length); /* j9munprotect.s */
```
While you're here, please remove the unused locals `index` and `unixFlags` declared on lines 45-46.

Reply:
addressed in https://github.com/eclipse-openj9/openj9/pull/19906/commits/c1b707335949fd2932b2058995595e7f6d170fcd